packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewerTextLayer.tsx

```diff
@@ -67,23 +68,29 @@ const PdfViewerTextLayer: FC<PdfViewerTextLayerProps> = ({
   const renderedText = useAsyncFunctionCall(
     useCallback(
       async (signal: AbortSignal) => {
-        if (textLayerDiv && loadedText) {
-          const { textContent, viewport, scale, page } = loadedText;
+        if (textLayerWrapper && loadedText) {
+          const { textContent, viewport, page } = loadedText;
 
           const builder = new TextLayerBuilder({
-            textLayerDiv,
-            viewport,
-            eventBus: new EventBus(),
-            pageIndex: page - 1
+            pdfPage: loadedPage
           });
+
+          textLayerWrapper.append(builder.div);
           signal.addEventListener('abort', () => builder.cancel());
 
-          await _renderTextLayer(builder, textContent, textLayerDiv, scale);
-          return { textContent, viewport, page, textDivs: builder.textDivs };
+          await _renderTextLayer(builder, textLayerWrapper, viewport);
+          //TODO: is textLayerWrapper the right value for the container?
+          const pdfJsTextLayer = new PdfJsTextLayer({
+            textContentSource: textContent,
+            container: textLayerWrapper,
+            viewport
+          });
```

Comment:
Main question to answer before this is mergable. TextLayer needs a `container`, and I am providing it with `textLayerWrapper`, which looks similar to what it's expecting. However, I'm seeing slightly different output in the `textDivs` styling this produces, so I'm checking whether that actually breaks anything and if we'll need to find a better `container`. Could also be a difference in the order in which I'm running renders that involve `textLayerWrapper`, since it gets instantiated inside `_renderTextLayer()` before I run the render on `PdfJsTextLayer`, which consumes it.